tools/test-utils.js

```diff
@@ -41,7 +41,16 @@ function getRaw(ast) {
  * @param {*} config the parser configuration
  * @returns {Function} callback for Jest test() block
  */
-function createSnapshotTestBlock(code, config) {
+function createSnapshotTestBlock(code, config = {}) {
+    const defaultConfig = {
+        loc: true,
+        range: true,
+        raw: true,
+        tokens: true,
+        comment: true,
```

Comment:
Not all the tests had `comment: true`, which is what is causing a lot of diffs in the snapshots. I personally think it's fine to have this always set.